test/MultiSig/recoverablemultisig.test.ts

```diff
@@ -0,0 +1,318 @@
+import { BigNumber } from 'bignumber.js';
+import { assert } from 'chai';
+
+import * as tempo from '@digix/tempo';
+import * as Web3 from 'web3';
+
+import {
+  RecoverableMultiSig,
+  RecoveryCancelledEvent,
+  RecoveryConfirmedEvent,
+  RecoveryStartedEvent,
+  SignHashArtifacts
+} from 'signhash';
+import { ContractContextDefinition } from 'truffle';
+
+import { toHex } from '../../multisig';
+
+import {
+  assertNumberEqual,
+  assertThrowsInvalidOpcode,
+  findLastLog
+} from '../helpers';
+import { MultiSigTestContext } from './context';
+
+declare const web3: Web3;
+declare const artifacts: SignHashArtifacts;
+declare const contract: ContractContextDefinition;
+
+const RecoverableMultiSigContract = artifacts.require(
+  './MultiSig/RecoverableMultiSig.sol'
+);
+
+contract('RecoverableMultiSigContract', accounts => {
+  describe.skip('#ctor', () => {
+    it('should not allow empty list of owners', async () => {
+      await assertThrowsInvalidOpcode(async () => {
+        await RecoverableMultiSigContract.new([]);
+      });
+    });
+  });
+
+  const ownerSets = [[accounts[2]], accounts.slice(1, 3), accounts.slice(2, 6)];
+  ownerSets.map(owners => {
+    context.skip(`When wallet has ${owners.length} owners`, () => {
```

Comment:
`skip`

Reply:
Skip usage already commented in other place